bindgen/raze/Cargo.toml

```diff
@@ -9,23 +9,25 @@ path = "fake_rules_rust_bindgen_lib.rs"
 bindgen = "0.55.1"
 libloading = "=0.6.3"
 
-[raze]
+[package.metadata.raze]
 genmode = "Remote"
 workspace_path = "//bindgen/raze"
 gen_workspace_prefix = "rules_rust_bindgen"
-output_buildfile_suffix = "BUILD"
+rust_rules_workspace_name = "io_bazel_rules_rust"
+package_aliases_dir = "."
+default_gen_buildrs = false
```

Comment:
I suspect these can be removed, and most of the per-package overrides removed, across this entire PR, but that can definitely be a follow-up :)

Reply:
Can you elaborate on what you mean here? What should get removed?

Reply:
We can probably leave `default_gen_buildrs` to its default of `true`, and stop worrying about it

Reply:
Ah, I see, yeah, that should be an explicit decision and didn't want to expand this PR with that change. I think a separate PR is the right thing to do 😄